### PR TITLE
fix tipo: `MHD_FEATURE_MESSGES` -> `MHD_FEATURE_MESSAGES`.

### PR DESCRIPTION
**Pull request by silvioprog**
Hello dude,

Please apply this PR. :-)

Thank you!

P.S: the example `timeout.c` was done by Grothoff and me in 2016. :-)

### CODE REVIEW

File: src/examples/timeout.c [hunk 1]
```diff
@@ -42,7 +66,7 @@ main (int argc,
                              PORT,
                              NULL, NULL,
                              &answer_to_connection, NULL,
-                             MHD_OPTION_CONNECTION_TIMEOUT, (unsigned int) 3,
+                             MHD_OPTION_CONNECTION_TIMEOUT, (unsigned int) 3, // 3 seconds
```
Comment (Karlson2k) [inline src/examples/timeout.c]: C++ style comment

**Comment by silvioprog** [comment]
You are one of the authors: https://lists.gnu.org/archive/html/libmicrohttpd/2015-12/msg00029.html . Please add you name on that too. Link [3] was the first version of this example, Christian and me improved that one in next messages at this topic.

File: src/examples/timeout.c (hunk 1 above)
Comment (silvioprog) [inline src/examples/timeout.c]: Could you remove it?

**Comment by silvioprog** [comment]
As soon as you apply or reject it I'll send more examples (gzip, etag..) I've suggested via bugtracker. :-)

**Comment by Karlson2k** [comment]
Let's not intermix different things in one PR.
According to PR name, it should fix typo. This commit already merged by cherry-pick.

To change copyright, I need approval from Christian as Christian created initial commit with this example.

**Comment by silvioprog** [comment]
You are absolutely right. Thanks for clarifying. I'm not expert in making PRs, so I'm going to send the copyright changes as patch via mailing list (I don't know why Github merged all changes in one PR :-/ ). However, I'll re-fork repo opening new PR sending a new sample showing how to translate MHD message errors (as issued in mantis [#0004924](https://gnunet.org/bugs/view.php?id=4924)) ...

**Comment by Karlson2k** [comment]
If you add more commits to the same branch - it will appear in PR. That's why people usually use individual specific branched for PRs. This will allow you to publish several PRs in the same time and update each one individually, if requited.
I'd recommend to move next discussions to https://gitlab.com/karlson2k/libmicrohttpd as GitLab is more friendly to software freedom.

**Comment by silvioprog** [comment]
Perfect, I'm going to Gitlab. :-) Thanks for the PR explanations and maintaining MHD updated in social GIT hosts. (new discussions and PR's will send via Gitlab .. :-) )

**Pull request closed without merging**